src/wrapper/java_vm/vm.rs

```diff
@@ -44,6 +44,11 @@ impl JavaVM {
         Ok(JavaVM(ptr))
     }
 
+    /// Returns underlying `sys::JavaVM` interface.
+    pub fn get_native_interface(&self) -> *mut sys::JavaVM {
+        self.0
```

Comment:
I wonder if it will benefit from integration tests (e.g., a roundtrip test), or is it too simple to break :-) ?

Reply:
Like, call `get_java_vm_pointer` follow by a `JavaVM::from_raw()`? And check that I can, for example, get back the env?

Reply:
Yes, I thought of something like that, but a test might get too complicated for such a simple getter.